### ISSUE TITLE
tronbox migrate error

### ISSUE THREAD

**Issue by satyakamc** [tronprotocol/tron-box#69]
HI, I am getting below error while migrating...
Satyakams-MacBook-Pro:CodeXpert-Tron-DApp-Template satyakamchakravarty$ tronbox migrate --reset --network shasta
(node:19151) V8: /usr/local/lib/node_modules/tronbox/build/cli.bundled.js:171304 Invalid asm.js: Invalid member of stdlib
Using network 'shasta'.

Running migration: 1_initial_migration.js
  Deploying Migrations...
ERROR: TypeError: Cannot read property 'code' of undefined
Satyakams-MacBook-Pro:CodeXpert-Tron-DApp-Template satyakamchakravarty$﻿

**Comment by sullof** [comment]
It looks like an error in the code.
If you could share it somehow, I can help you better.

**Comment by cma00000** [comment]
Same Error Here. with network setting as follow:

```
    shasta: {
      privateKey: process.env.PK,
      consume_user_resource_percent: 30,
      fee_limit: 100000000,
      fullNode: "https://api.shasta.trongrid.io",
      solidityNode: "https://api.shasta.trongrid.io",
      eventServer: "https://api.shasta.trongrid.io",
      network_id: "*"
    },
```

**Comment by cma00000** [comment]
And have following error logs using tron studio to deploy contract (hope it may help to target the problem) @satyakamc 

```
2018-12-26 15:10:56.364 java[17289:24925007] unrecognized type is 4294967295
2018-12-26 15:10:56.364 java[17289:24925007] *** Assertion failure in -[NSEvent _initWithCGEvent:eventRef:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1671/AppKit.subproj/NSEvent.m:1969
```

**Comment by taiyangc** [comment]
The author's issue should have been resolved by upgrading to tronbox@2.3.4 (it's an account lacking bandwidth error). Are you facing the same issue with 2.3.4 @cma00000 ?

**Comment by cma00000** [comment]
It is the same lacking bandwidth issue. Thank you all for help.
